mage_ai/cluster_manager/aws/ecs_task_manager.py

```diff
@@ -0,0 +1,111 @@
+from typing import List
+from botocore.config import Config
+from mage_ai.services.aws.ecs.ecs import create_environment_task, list_tasks
+from mage_ai.shared.array import find
+
+import boto3
+import os
+
+from mage_ai.shared.hash import dig
+
+
+CLUSTER_NAME = 'mage-data-prep-development-cluster'
+
+class EcsTaskManager():
+    def __init__(self, cluster_name=CLUSTER_NAME):
+        self.cluster_name = cluster_name
+
+    def list_tasks(self):
+        region_name = os.getenv('AWS_REGION_NAME', 'us-west-2')
+        config = Config(region_name=region_name)
+        ec2_client = boto3.client('ec2', config=config)
+        response = list_tasks(self.cluster_name)['tasks']
+
+        network_interfaces = self.__get_network_interfaces(response, ec2_client)
+
+        tasks = []
+
+        for index, task in enumerate(response):
+            public_ip = dig(network_interfaces[index], 'Association.PublicIp')
+
+            tasks.append(dict(
+                ip=public_ip,
+                group=task['group'],
+                status=task['lastStatus'],
+                type=task['launchType'],
+            ))
+
+        return tasks
+
+    def create_task(self, name: str):
+        region_name = os.getenv('AWS_REGION_NAME', 'us-west-2')
+        config = Config(region_name=region_name)
+        ec2_client = boto3.client('ec2', config=config)
+        ecs_client = boto3.client('ecs', config=config)
+
+        # create new task definition
+        base_td = \
+            ecs_client.describe_task_definition(
+                taskDefinition='mage-dev-dev-task'
+            )['taskDefinition']
+        remove_args = [
+            'compatibilities',
+            'registeredAt',
+            'registeredBy',
+            'requiresAttributes',
+            'revision',
+            'status',
+            'taskDefinitionArn',
+        ]
+        for arg in remove_args:
+            base_td.pop(arg)
+
+        container_path = f'/home/src/{name}'
```

Comment:
we don't need to update the container path here. we can mount EFS to the same container path and just use the different folders in the FS.